source/reference/minio-mc/mc-batch-start.rst

```diff
@@ -0,0 +1,117 @@
+.. _minio-mc-batch-start:
+
+==================
+``mc batch start``
+==================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc batch start
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+Syntax
+------
+
+.. start-mc-batch-start-desc
+
+The :mc:`mc batch start` command launches a batch job from a job batch YAML file.
+
+.. end-mc-batch-start-desc
+
```

Comment:
It occurs to me that a batch is one-time vs replication - we should specify that here in short and expand on it in batch-framework.rst

Reply:
Need to keep this file generic, though, as down the road it may be more than just replication. I'll chew on it and work out something.

Reply:
Done.